### PR TITLE
Update `ffmpeg` spec

### PR DESCRIPTION
**Pull request by ryanccn**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**

regular update for `ffmpeg`

**What is the current behavior? (You can also link to an open issue here)**

**What is the new behavior (if this is a feature change)?**

**Additional info:**

### CODE REVIEW

File: src/ffmpeg.ts [hunk 1]
```diff
@@ -1,11 +1,12 @@
 /**
  * Generated by ffmpeg-fig at https://github.com/ryanccn/ffmpeg-fig
- * Wed, 09 Feb 2022 11:34:45 GMT
+ * Thu, 08 Sep 2022 14:13:16 GMT
  */
 
 const completionSpec: Fig.Spec = {
   name: "ffmpeg",
   description: "Play, record, convert, and stream audio and video",
+
```
Comment (SeparateRecords) [inline src/ffmpeg.ts:9]: This PR introduces a lot of blank lines, can you remove these?
If you're a Vim user: `:%s/,\n\n/,\r/g`

**Review by SeparateRecords — CHANGES_REQUESTED** [review]
Looking good! One small comment to resolve then it's an easy merge

File: src/ffmpeg.ts (hunk 1 above)
Comment (mschrage) [reply src/ffmpeg.ts:9]: Or perhaps just update the function that generates this spec?
https://github.com/ryanccn/ffmpeg-fig

Comment (ryanccn) [reply src/ffmpeg.ts:9]: Well fair, probably some new line issues in the template. I'll fix it this weekend hopefully!

Comment (ryanccn) [reply src/ffmpeg.ts:9]: Quit a bit late but it's fixed :D

File: src/ffmpeg.ts [hunk 2]
```diff
@@ -1,11 +1,12 @@
 /**
  * Generated by ffmpeg-fig at https://github.com/ryanccn/ffmpeg-fig
- * Wed, 09 Feb 2022 11:34:45 GMT
+ * Tue, 11 Oct 2022 12:43:14 GMT
  */
 
-const completionSpec: Fig.Spec = {
+ const completionSpec: Fig.Spec = {
```
Comment (SeparateRecords) [inline src/ffmpeg.ts:6]:
```suggestion
const completionSpec: Fig.Spec = {
```

**Review by SeparateRecords — CHANGES_REQUESTED** [review]
LGTM! Only one change, I think the linter is failing specifically because of this one line

**Comment by SeparateRecords** [comment]
(Quick fix: `npx prettier -w src/ffmpeg.ts`)

**Comment by SeparateRecords** [comment]
Also, if you want to close and reopen the PR for the hacktoberfest credit, I wouldn't blame you!! You're welcome to 🙂

**Review by SeparateRecords — APPROVED** [review]
Awesome, thanks so much @ryanccn!!

**Pull request merged**